### ISSUE TITLE
Snippet doesn't get triggered for certain tokens

### ISSUE THREAD

**Issue by Herasselbach** [microsoft/vscode#124751]
- VS Code Version: 1.56.2

- OS Version:  MacOS BigSur 11.2.3

Steps to Reproduce:
I'm using VSCode for editing latex files and have created certain snippet in the `latex.json` file. Most of them work, but for certain characters those snippet don't get triggered:

```
"implies": {
        "prefix": ["=>"],
        "body": [
            "\\Longrightarrow",
        ],
        "description": "implies"
    },
```
This is one example, the snippet doesn't appear when typing `=>`. Other characters where it didn't work for me are:
`=<`, `/`, '!=`.

**Comment by jrieken** [comment]
>  the snippet doesn't appear when typing =>

I assume you refer to quick suggestion aka suggestion as you type, right? Does this work when you press Ctrl+Space to trigger suggestions explicitly?

**Comment by Herasselbach** [comment]
@jrieken yes, it get's shown when I press ctrl+space explicitly but doesn't show up automatically.

**Comment by Herasselbach** [comment]
@jrieken so how should I now proceed?

**Comment by jrieken** [comment]
You can reach out to author of the latex extension and/or wait for use relax how quick suggestions are triggered (which isn't currently planned)

**Comment by Herasselbach** [comment]
@jrieken  can I somehow modify the extension on my own in order to achieve the desired behaviour?
